docs/fields.md

```diff
@@ -14,24 +14,27 @@ supported in **all field types**.
 - `primary_key` - A boolean. Determine if a column is primary key.
 Check the [primary_key](./models.md#restrictions-with-primary-keys) restrictions with Edgy.
 - `exclude` - An bool indicating if the field is included in model_dump
-- `default` - A value or a callable (function).
 - `index` - A boolean. Determine if a database index should be created.
 - `inherit` - A boolean. Determine if a field can be inherited in submodels. Default is True. It is used by PKField, RelatedField and the injected ID Field.
 - `skip_absorption_check` - A boolean. Default False. Dangerous option! By default when defining a CompositeField with embedded fields and the `absorb_existing_fields` option it is checked that the field type of the absorbed field is compatible with the field type of the embedded field. This option skips the check.
 - `unique` - A boolean. Determine if a unique constraint should be created for the field.
 Check the [unique_together](./models.md#unique-together) for more details.
+- `column_name` - A string. Database name of the column (by default the same as the name)
+- `comment` - A comment to be added with the field in the SQL database.
+- `secret` - A special attribute that allows to call the [exclude_secrets](./queries/secrets.md#exclude-secrets) and avoid
+accidental leakage of sensitive data.
 
-All the fields are required unless on the the following is set:
+All fields are required unless one of the following is set:
 
 - `null` - A boolean. Determine if a column allows null.
 
     <sup>Set default to `None`</sup>
 
 - `server_default` - instance, str, Unicode or a SQLAlchemy `sqlalchemy.sql.expression.text`
 construct representing the DDL DEFAULT value for the column.
-- `comment` - A comment to be added with the field in the SQL database.
-- `secret` - A special attribute that allows to call the [exclude_secrets](./queries/secrets.md#exclude-secrets) and avoid
-accidental leakage of sensitive data.
+- `default` - A value or a callable (function).
```

Comment:
We want to have the comment. It's a database property anyway. No need to remove it and it's backwards compatible.

Reply:
I only moved the comment docs up. I didn't remove it.